src/main/java/transforms/NestedTransformsTransform.java

```diff
@@ -0,0 +1,37 @@
+package transforms;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.openlattice.shuttle.transformations.TransformValueMapper;
+import com.openlattice.shuttle.transformations.Transformation;
+import com.openlattice.shuttle.util.Constants;
+
+import java.util.List;
+import java.util.Map;
+
+public class NestedTransformsTransform extends Transformation<Map<String, Object>> {
+
+    private final TransformValueMapper transformValueMapper;
+
+    /**
+     * Represents a series of transformations to be used as an argument (nested within other transformations)
+     *
+     * @param transforms: transforms
+     */
+    @JsonCreator
+    public NestedTransformsTransform(
+            @JsonProperty( Constants.TRANSFORMS ) List<Transformation> transforms
+    ) {
+        this.transformValueMapper = new TransformValueMapper(transforms);
```

Comment:
I'm a little confused how this is different than just using TransformValueMapper

Reply:
Madness.

Reply:
TransformValueMapper is `SerializableFunction<Map<String, Object>, Object>`
Transformation is `Function<I, Object>`
So TransformValueMapper can't be input for a propertyDefinition.

So this is kind of like a wrapper/parser to put transformations in the TransformValueMapper...  What would be a better strategy?

Other than that it's really ugly and bugprone in the flights to have the full path:
```
- !<com.openlattice.shuttle.transformations.TransformValueMapper>
```
vs.
```
- !<transforms.NestedTransformsTransform>
```

Reply:
Is this situation fixed by just changing `Transformation` to implement `SerializableFunction` instead of `Function` ?

Reply:
@jokedurnez ping!

Reply:
I'll just close this.  From the comments it seems like this request is ridiculous.
(I can't find a SerializableFunction, and Serializable doesn't have an apply function in the superclass)